CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # bedrock-ledger ChangeLog
 
+## 11.0.1 - 2020-TBD
+
+### Changed
+- Remove `bedrock-identity` and use `bedrock-account` in tests.
+
```

Comment:
I think we shouldn't even report any changes here -- they are internal to the test suite. There will be no new release on this basis and it doesn't affect any public facing users of the library.

```suggestion
```

Reply:
Once this is removed, I'll merge this PR. Thanks!